Plugins/Sycl/include/Acts/Plugins/Sycl/Seeding/CreateSeedsForGroupSycl.hpp

```diff
@@ -16,11 +16,17 @@
 
 namespace Acts::Sycl {
 
-/// @brief Seedfinding algorithm implemented with SYCL to offload computations
-/// to GPUs.
+/// @brief Seedfinding algorithm implemented in SYCL.
+///
+/// @param [in] wrappedQueue is a wrapper object of the SYCL queue
+/// @param [in] seedfinderConfig includes the required configuration
+/// parameters for the algorithm
+/// @param[in] deviceCuts is an experiment specific object with customizable
+/// seed weight altering and seed cutting member functions
+/// @param [out] seeds holds of the generated seed indices and weight
```

Comment:
What about `{bottom,middle,top}SPs`?

Reply:
I know, I left them out because I thought they don't need further explanation. I'll just add comments for them too, so it's more consistent. Thanks. :)